fairroot/base/event/FairTimeStamp.h

```diff
@@ -74,7 +74,7 @@ class FairTimeStamp : public FairMultiLinkedData_Interface
         return out;
     }
 
-    virtual bool operator<(const FairTimeStamp* rValue) const { return GetTimeStamp() < rValue->GetTimeStamp(); }
+    bool operator<(const FairTimeStamp& rValue) const { return GetTimeStamp() < rValue.GetTimeStamp(); }
```

Comment:
1. Please update the copyright year in the header

2. This is a breaking change, because it drops an old API. Should we document this in the Chanelog?

3. I don't really understand, why we use the getters instead of accessing the member variables directly?

   ```suggestion
       bool operator<(const FairTimeStamp& rValue) const { return fTimeStamp < rValue.fTimeStamp; }
   ```

4. Should this also compare the timestamp-error, like the `Compare` method above does?

Reply:
1. done
2. What is the opionion of the others?
3. Variables are now directly used
4. No. The important comaprison is with the time stamp. An additional comparison with the error does not realy make sense.

Reply:
About (4): The Scroll a bit up in the file `Compare` compares the error. So that method and this operator have different results? IMHO we should at least document this difference.